libs/ngrx-toolkit/src/lib/with-storage-sync.spec.ts

```diff
@@ -1,14 +1,16 @@
 import { getState, patchState, signalStore, withState } from '@ngrx/signals';
-import { withStorageSync } from './with-storage-sync';
+import { withBigIntRehydration, withBigIntReplacement, withStorageSync } from './with-storage-sync';
 import { TestBed } from '@angular/core/testing';
 
 interface StateObject {
   foo: string;
   age: number;
+  big: bigint;
```

Comment:
Would it be possible, that we leave the tests as they are and just create a separate test which checks the de/serialization for `bigint`?